book.rb

```diff
@@ -0,0 +1,12 @@
+require './rental'
+
+class Book
```

Comment:
- [ ] A method for adding rentals to `@rentals` should be present in order to make this class functional, please add it. I'd suggest taking a look at [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).